lib/packagedata.lib

```diff
@@ -45,7 +45,7 @@ getAppname() {
 
   # still here, so not found in cache. Try AAPT (if we have it)
   if [[ $HAVE_AAPT -eq 1 ]]; then
-    local str="$(adb ${ADBOPTS} shell "aapt d badging $2 2>/dev/null | grep 'application: label' | sed -n \"s/.*label\='\([^']*\)'.*/\1/p\"")" # label='Some App'"
+    local str="$(adb ${ADBOPTS} shell "aapt d badging $2 2>/dev/null" | grep 'application: label' | sed -n "s/.*label\='\([^']*\)'.*/\1/p")" # label='Some App''
```

Comment:
good idea. If `sed` is missing on the PC end, it's probably much easier to add. :white_check_mark: